pylutron/__init__.py

```diff
@@ -716,6 +721,29 @@ def is_dimmable(self):
     return self.type != 'NON_DIM' and not self.type.startswith('CCO_')
 
 
+class Shade(Output):
+  """This is the output entity for shades in Lutron universe."""
+  switched/dimmed load, e.g. light fixture, outlet, etc."""
+  _ACTION_RAISE = 2
```

Comment:
should we do something with the tilt and lift things too? Do those exist for your shades too? I wish I had some Sivoias to play with.

Reply:
I don't even have shades.  And this is going to get hideous.  I really need the XML from someone that has both types of shades to figure everything out.  The API for the different types of shades is completely different, making me want to go educate Lutron on how to write code.

Reply:
Yeah, i don't have shades either LOL.

I wish someone who has shades on Radio Ra 2 could give us some XML magic :)

Reply:
So I think I've pushed this as far as I can without having shades.  The motivation was to attempt to resolve a long outstanding HomeAssistant feature request for shades and figured it would be easy enough to throw this in and see if that makes people happy.

Reply:
@cdheiser:  I have the shades and a RR2 system that I'd be happy to grab the XML from.  I'm having an issue w/ the HA lutron integration as well as my homes have multiple main repeaters.  The integration works flawlessly in one home, and it doesn't work at all in the other.  Would love some help with that issue too! :)